arcgis-ios-sdk-samples/Maps/Display device location with NMEA data sources/DisplayDeviceLocationWithNMEADataSourcesViewController.swift

```diff
@@ -99,7 +99,7 @@ class DisplayDeviceLocationWithNMEADataSourcesViewController: UIViewController {
             preferredStyle: .actionSheet
         )
         // Add real data source to the options.
-        let realDataSourceAction = UIAlertAction(title: "Device", style: .default) { [self] _ in
+        let realDataSourceAction = UIAlertAction(title: "Device", style: .default) { [unowned self] _ in
```

Comment:
I changed this to unowned because I think it is more correct, as the action has the same/shorter lifespan as the `UIBarButtonItem`.